app/controllers/healthcheck_controller.rb

```diff
@@ -0,0 +1,29 @@
+# rubocop:disable Rails/ApplicationController
+class HealthcheckController < ActionController::Base
+  def show
+    render json: { healthcheck: healthcheck_result }, status: http_status
+  end
+
+  private
+
+  def healthcheck_result
+    green? ? 'OK' : 'NOT OK'
+  end
+
+  def http_status
+    green? ? 200 : 503
+  end
+
+  def green?
+    # add more checks as needed:
+    # database_connected? && redis_connected etc...
+    database_connected?
+  end
+
+  def database_connected?
+    ApplicationRecord.connection.select_value('SELECT 1') == 1
```

Comment:
I think we can use `ActiveRecord::Base.connection.active?` for the same purpose, that's what I've used in the past, although unsure what is more optimal or recommended tho 🤔

Reply:
That sounds cleaner to me I'll amend 👍